prog/weave-kube/weave-daemonset-k8s-1.6.yaml

```diff
@@ -1,120 +1,167 @@
-kind: ClusterRole
-apiVersion: rbac.authorization.k8s.io/v1beta1
-metadata:
-  name: weave-net
-rules:
-- apiGroups:
-  - ""
-  resources:
-  - pods
-  - namespaces
-  - nodes
-  verbs:
-  - get
-  - list
-  - watch
-- apiGroups:
-  - extensions
-  resources:
-  - networkpolicies
-  verbs:
-  - get
-  - list
-  - watch
----
 apiVersion: v1
-kind: ServiceAccount
-metadata:
-  name: weave-net
-  namespace: kube-system
----
-kind: ClusterRoleBinding
-apiVersion: rbac.authorization.k8s.io/v1beta1
-metadata:
-  name: weave-net
-roleRef:
-  apiGroup: rbac.authorization.k8s.io
-  kind: ClusterRole
-  name: weave-net
-subjects:
-- kind: ServiceAccount
-  name: weave-net
-  namespace: kube-system
----
-apiVersion: extensions/v1beta1
-kind: DaemonSet
-metadata:
-  name: weave-net
-  namespace: kube-system
-spec:
-  template:
-    metadata:
+kind: List
+items:
+  - metadata:
       labels:
         name: weave-net
+      name: weave-net
+      annotations:
+        cloud.weave.works/launcher-info: |-
+          {
+            "server-version": "master-c1f3803",
+            "original-request": {
+              "url": "/k8s/v1.6/net?v=latest",
+              "date": "Thu Apr 20 2017 09:43:20 GMT+0000 (UTC)"
+            },
+            "email-address": "support@weave.works"
+          }
+    apiVersion: v1
+    kind: ServiceAccount
```

Comment:
This is the `kind` of the first item in the list?  Seems weird it comes at the end.